docs/marketplace/dev/feature-walkthroughs/202108.0/marketplace-order-management-feature-walkthrough.md

```diff
@@ -1,44 +1,95 @@
 {% info_block warningBox "User documentation" %}
 
-To learn more about the feature and to find out how end users use it, see [Marketplace Order Management](/docs/marketplace/user/features/{{page.version}}/marketplace-order-management-feature-overview/marketplace-order-management-feature-overview.html) feature overview for business users.
+Merchant Order is created by Merchant State Machine in MerchantStateMachineHandlerPlugin, it requires StateMachineItem and MerchantOrderItem
 
+To learn more about the feature and to find out how end users use it, see [Marketplace Order Management](/docs/marketplace/user/features/{{page.version}}/marketplace-order-management-feature-overview/marketplace-order-management-feature-overview.html) feature overview for business users.
 {% endinfo_block %}
 
-## Module relations
+## Module dependency graph
 
-* Relations between modules in the Marketplace Order entity:
+Merchant Order
+![Module Dependency Graph](https://confluence-connect.gliffy.net/embed/image/901b201a-030b-4824-a136-ef06d258a41b.png?utm_medium=live&utm_source=confluence)
 
-![Marketplace order module relations](https://spryker.s3.eu-central-1.amazonaws.com/docs/Marketplace/dev+guides/Feature+walkthroughs/Marketplace+Order+Management/Marketplace+order+module+relations.png)
+<!--
+Diagram content:
+    -The module dependency graph SHOULD contain all the modules that are specified in the feature  (don't confuse with the module in the epic)
+    - The module dependency graph MAY contain other module that might be useful or required to show
+Diagram styles:
+    - The diagram SHOULD be drown with the same style as the example in this doc
+    - Use the same distance between boxes, the same colors, the same size of the boxes
+Table content:
+    - The table that goes after diagram SHOULD contain all the modules that are present on the diagram
+    - The table should provide the role each module plays in this feature
+-->
+| MODULE     | DESCRIPTION                |
+|------------|----------------------------|
+| MerchantOrderTotal | Provides a sum calculation for a merchant order|
+| MerchantOrder | Provides Merchant related information for Order |
+| MerchantOrderItem | Provides Merchant related information for Order Item |
 
-* Relations between modules in the Merchant Order entity:
 
-![Merchant order module relations](https://spryker.s3.eu-central-1.amazonaws.com/docs/Features/Marketplace/Marketplace+and+Merchant+orders/Merchant+order+feature+overview/merchant-order-module-relations.png)
+## Domain model
+<!--
+- Domain model SHOULD contain all the entities that were adjusted or introduced by the feature.
+- All the new connections SHOULD also be shown and highlighted properly 
+- Make sure to follow the same style as in the example
+-->
+![Domain Model](https://confluence-connect.gliffy.net/embed/image/041ca5e4-7738-47ac-a01b-4ed91a57662d.png?utm_medium=live&utm_source=confluence)
```

Comment:
This diagram is not accurate, please show real entities with real dependecies.
1. ProductOffer has no connection to Orders, but has to OrderItem
2. We don't have MarketplaceOrders but SalesOrders 
3. We don't have MerchantSalesOrderStatus
4. We don't have OmsMerchantOrderProcess

Please adjust.